archeogitvsszz/vulnerabilities.py

```diff
@@ -1,14 +1,49 @@
 import logging
+from os import listdir
+from os.path import isfile, join
 
 logger = logging.getLogger(__name__)
 
 
 class Vulnerabilities:
     def __init__(self, path):
         self._path = path
+        self._cve_path = join(path, "cves")
+        self._cves = self.get_all_file_names()
+        self._index = 0
 
     def __iter__(self):
-        pass
+        return iter(self._cves)
 
     def __next__(self):
-        pass
+        self._index += 1
```

Comment:
Wouldn't incrementing `self._index` before getting the current value at `self._index` miss returning the first element?

Reply:
Hmm, this is my first time using `__next__`, but it appears that all of the cves were processed.